google/cloud/bigquery/job/load.py

```diff
@@ -277,6 +277,27 @@ def null_marker(self):
     def null_marker(self, value):
         self._set_sub_prop("nullMarker", value)
 
+    @property
+    def projection_fields(self):
+        """Optional[List[str]]: If
+        :attr:`google.cloud.bigquery.job.LoadJobConfig.source_format` is set to
+        "DATASTORE_BACKUP", indicates which entity properties to load into
+        BigQuery from a Cloud Datastore backup.
+
+        Property names are case sensitive and must be top-level properties. If
+        no properties are specified, BigQuery loads all properties. If any
+        named property isn't found in the Cloud Datastore backup, an invalid
+        error is returned in the job result.
+
+        See:
+        https://cloud.google.com/bigquery/docs/reference/rest/v2/Job#JobConfigurationLoad.FIELDS.projection_fields
+        """
+        return self._get_sub_prop("projectionFields")
+
+    @projection_fields.setter
+    def projection_fields(self, value):
+        self._set_sub_prop("projectionFields", value)
```

Comment:
Sanity check - if `value` is `None`, should we omit the `projectionFields` key altogether? Or do we still want it to be present and explicitly set to `None`?

(I'm never quite sure when this makes a difference on the backend and when not)

Reply:
I don't think it should make a difference in this case since jobs aren't mutable. In general, I prefer to send explicit `None` if we got one, since that allows things like "unsetting" values.

Reply:
Makes sense. :+1: